<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1335</width>
    <height>707</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="current_picture">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>80</y>
     <width>1051</width>
     <height>521</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="previous_photo">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="next_photo">
   <property name="geometry">
    <rect>
     <x>1220</x>
     <y>320</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonInfo">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>10</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonModify">
   <property name="geometry">
    <rect>
     <x>1110</x>
     <y>10</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonThrow">
   <property name="geometry">
    <rect>
     <x>1150</x>
     <y>10</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="tagsWidget">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>620</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="backgroundVisible">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
